source/objects/constraintClass.m

```diff
@@ -126,34 +126,34 @@ function setInitDisp(obj, x_rot, ax_rot, ang_rot, addLinDisp)
             %``addLinDisp`` ('float') is the initial linear displacement [m] in addition to the displacement caused by the constraint rotation, Default = '[]'.
             loc = obj.loc;
             relCoord = loc - x_rot;
-            rotatedRelCoord = obj.rotateXYZ(relCoord,ax_rot,ang_rot);
+            rotatedRelCoord = rotateXYZ(relCoord,ax_rot,ang_rot);
             newCoord = rotatedRelCoord + x_rot;
             linDisp = newCoord-loc;
             obj.initDisp.initLinDisp= linDisp + addLinDisp; 
         end
 
-        function xn = rotateXYZ(obj,x,ax,t)
-            %This method rotates a point about an arbitrary axis.
-            %
-            %``x`` (`3x1 float vector`) is the point coordiantes.
-            %
-            %``ax`` (`3x1 float vector`) is the axis about which to rotate the constraint and must be a normal vector.
-            %
-            %``t``  (`float`) is the rotation angle of the constraint.
-            % 
-            %``xn`` (`3x1 float vector`) is the new point coordiantes after rotation.
-            rotMat = zeros(3);
-            rotMat(1,1) = ax(1)*ax(1)*(1-cos(t))    + cos(t);
-            rotMat(1,2) = ax(2)*ax(1)*(1-cos(t))    + ax(3)*sin(t);
-            rotMat(1,3) = ax(3)*ax(1)*(1-cos(t))    - ax(2)*sin(t);
-            rotMat(2,1) = ax(1)*ax(2)*(1-cos(t))    - ax(3)*sin(t);
-            rotMat(2,2) = ax(2)*ax(2)*(1-cos(t))    + cos(t);
-            rotMat(2,3) = ax(3)*ax(2)*(1-cos(t))    + ax(1)*sin(t);
-            rotMat(3,1) = ax(1)*ax(3)*(1-cos(t))    + ax(2)*sin(t);
-            rotMat(3,2) = ax(2)*ax(3)*(1-cos(t))    - ax(1)*sin(t);
-            rotMat(3,3) = ax(3)*ax(3)*(1-cos(t))    + cos(t);
-            xn = x*rotMat;
-        end
+%         function xn = rotateXYZ(obj,x,ax,t)

```

Comment:
Repeating comment, but suggest removing the commented lines before final push to the dev branch.